frontend/javascripts/admin/user/user_list_view.tsx

```diff
@@ -371,8 +370,8 @@ function UserListView({ activeUser, activeOrganization }: Props) {
         <div className="clearfix" />
       </div>
 
-      {isUserInvitesDisabled ? renderUpgradePlanAlert() : null}
-      {noOtherUsers && !isUserInvitesDisabled ? renderInviteUsersAlert() : null}
+      {isNewUserInvitesDisabled ? renderUpgradePlanAlert() : null}
+      {noOtherUsers && !isNewUserInvitesDisabled ? renderInviteUsersAlert() : null}
```

Comment:
Should these messages be adapted to advertise the possibility to invite guests, so users that other organizations already pay for?